src/khash.h

```diff
@@ -137,11 +137,7 @@ typedef unsigned int khint32_t;
 typedef unsigned long khint32_t;
 #endif
 
-#if ULONG_MAX == ULLONG_MAX
-typedef unsigned long khint64_t;
-#else
-typedef unsigned long long khint64_t;
-#endif
+typedef int64_t khint64_t;
```

Comment:
I hope this simplification is OK.

Reply:
We've already deviated from `khash.h` and I think that it's basically unmaintained, so I'm not too worried about deviating from upstream.  Therefore, I'm definitely okay with dropping the indirection around whether `ULONG_MAX == ULLONG_MAX` - that definitely makes sense.  But does this need to be `uint64_t` instead of `int64_t`?  I haven't even glanced at the arithmetic in the file, so it may be obvious that it is safe to move it to signed, but that's what jumps out at me from a cursory glance.

Reply:
Oops, you're right! I missed that.